### PR TITLE
add measures

### PR DESCRIPTION
**Pull request by andrewscolm**
First pass at adding "ever had a test". 
* ever had a positive test in sgss
* ever had any evidence of positive test or covid (primary care / SGSS / hospitalisation) 
* death not included as it is restricted to registered patients each month.

### CODE REVIEW

File: project.yaml [hunk 1]
```diff
@@ -45,9 +45,31 @@ actions:
       moderately_sensitive:  
         figs: output/figs.svg
 
+  generate_study_population_1:
+    run: cohortextractor:latest generate_cohort --study-definition study_definition_monthly --index-date-range "2020-03-01 to 2021-02-01 by month" --output-format feather
+    outputs:
+      highly_sensitive:
+        cohort: output/monthly/input_monthly_*.feather
+
+  generate_study_population_2:
```
Comment (wjchulme) [inline project.yaml:54]: Just to confirm, are these being split up to avoid memory problems?

**Review by wjchulme — COMMENTED** [review]
Just minor change needed but happy for this to be fixed and run

File: project.yaml (hunk 1 above)
Comment (andrewscolm) [reply project.yaml:54]: Yea, that's right. Are you happy with 2020-03-01 as the start date? I will extend it up to May 2022.

File: project.yaml [hunk 2]
```diff
@@ -45,9 +45,31 @@ actions:
...
+      outputs:
+        highly_sensitive:
+          cohort: output/monthly/input_monthly*.feather
+
+  generate_measures:
+    run: >
+      cohortextractor:latest generate_measures
+        --study-definition study_definition_monthly
+        --output-dir output/monthly
+    needs: [generate_study_population_1]
```
[the remark this answers is not in the document]
Comment (andrewscolm) [reply project.yaml]: Good spot, thanks.

**Pull request merged**